### PR TITLE
Feat rspy361/taints

### PR DESCRIPTION
**Pull request by eroan-marie**

### CODE REVIEW

File: inventory/sample/cluster.tfvars [hunk 1]
```diff
@@ -53,4 +61,5 @@ eip_elb_type = "5_bgp"
 eip_elb_bandwidth = 10
 
 #Bucket vars
-buckets = ["elasticsearch-processing", "elasticsearch-security", "thanos", "loki", "tempo", "psql"]
\ No newline at end of file
+#buckets = ["elasticsearch-processing", "elasticsearch-security", "thanos", "loki", "tempo", "psql"]
+buckets = ["test-ansible-collection"]
```
Comment (nleconte-csgroup) [inline inventory/sample/cluster.tfvars]: Are you sure about this one ?

File: inventory/sample/cluster.tfvars
Comment (nleconte-csgroup) [inline inventory/sample/cluster.tfvars:1]: Could you also add the new node group in the template ?

**Review by nleconte-csgroup — CHANGES_REQUESTED** [review]

File: inventory/sample/cluster.tfvars (hunk 1 above)
Comment (eroan-marie) [reply inventory/sample/cluster.tfvars]: Oups, it was on the bottom of the file, I didn't see it. 
It's fixed :smile:

File: inventory/sample/cluster.tfvars
Comment (eroan-marie) [reply inventory/sample/cluster.tfvars:1]: It's done

**Review by nleconte-csgroup — APPROVED** [review]

**Pull request merged**